packages/mdc-select/_mixins.scss

```diff
@@ -272,7 +272,7 @@
     }
 
     > option {
-      @include mdc-theme-prop(background-color, surface);
+      @include mdc-theme-prop(background-color, surface, $edgeOptOut: true);
```

Comment:
@williamernest Do you know why we're applying styles to native elements here? select / option native elements doesn't allow style customization in most of the browser as far as I know.